pkg/log/logger.go

```diff
@@ -82,3 +171,87 @@ func GetZapConfigs() *zap.Config {
 	_ = Logger()
 	return zapConfigs
 }
+
+// SetDefaultLogger allows customization of the default logger
+func SetDefaultLogger(handle *LoggerHandle) {
+	once.Do(initLogger)
+	defaultLogger.Store(handle)
+}
+
+// UpdateLoggingConfig is used to reconfigure logging.
+// This uses config keys of the form log.{logger}.level={level}.
+// The default level is set by log.level={level}
+func UpdateLoggingConfig(config map[string]string) {
+	once.Do(initLogger)
+	initLoggingConfig(config)
+}
+
+func initLoggingConfig(config map[string]string) {
+	levelMap := make(map[string]zapcore.Level)
+	levelMap[""] = zapcore.InfoLevel
```

Comment:
Can we have a constant for `""`, it's not entirely clear what it represents. I guess it's a default level, but a naming would help.

Reply:
Will update to `nullLogger` in next PR.